app/frontend/wallet/cardano-wallet.js

```diff
@@ -162,29 +162,37 @@ const CardanoWallet = async (options) => {
     return await blockchainExplorer.fetchTxInfo(txHash)
   }
 
+  function dropUnprofitableUtxos(utxos) {
+    const profitableUtxos = []
+    const witnessSize = 139
```

Comment:
this should be moved probably to constants.js and I would call it "txWitnessByteSize"